### PR TITLE
Add back f1 support (deprecated). Closes #231

### LINKED ISSUE #231: Support f1 addresses as well

### PR DESCRIPTION
**Pull request by juliangruber**
Closes #231

Temporary fix while Station Desktop doesn't yet support 0x/f4 addresses.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,12 +70,11 @@ the configuration options described in
 [Common Configuration](#common-configuration):
 
 - `FIL_WALLET_ADDRESS` _(string; required)_: Address of the Filecoin wallet that
-  will receive rewards. The value must be a mainnet address starting with `f410`
-  or `0x`.
+  will receive rewards. The value must be a mainnet address starting with
+  `f410`, `0x` (or `f1`, deprecated).
```
Comment (bajtos) [inline README.md:74]:
```suggestion
  will receive rewards. The value must be a mainnet address starting with
  `f410`, `0x`. 
  
  Addresses starting with `f1` are deprecated. We allow them
  for now but will start rejecting them in an upcoming version.
```

File: commands/station.js [hunk 2]
```diff
@@ -20,7 +20,12 @@ export const station = async ({ json, experimental }) => {
     console.error('FIL_WALLET_ADDRESS required')
     process.exit(1)
   }
-  if (!FIL_WALLET_ADDRESS.startsWith('f410') && !FIL_WALLET_ADDRESS.startsWith('0x')) {
+  if (FIL_WALLET_ADDRESS.startsWith('f1')) {
+    console.error('Warning: f1... addresses are deprecated')
```
Comment (bajtos) [inline commands/station.js:24]: A good error message is actionable - it should tell the user how to fix the problem or at least what to do next.

```suggestion
    console.error('Warning: f1... addresses are deprecated and will not receive any rewards.')
    console.error('Please use an address starting with f410 or 0x')
```

**Review by bajtos — APPROVED** [review]
LGTM with a suggestion on how to improve the error message.

Thank you for taking care of this!

**Comment by juliangruber** [comment]
https://github.com/filecoin-station/core/releases/tag/v14.4.0

**Pull request merged**